website/versioned_docs/version-0.65/customizing-SDK-versions.md

```diff
@@ -0,0 +1,97 @@
+---
+id: version-0.65-customizing-sdk-versions
+title: Customizing SDK versions
+original_id: customizing-sdk-versions
+---
+
+In React Native for Windows 0.64.3, we have made it easier for an app to customize which versions of the Windows SDK and WinUI 2.x to use.
```

Comment:
Maybe rephrase this now that this is for 65? "It is easy for an app to customize..." or something like that